test/integration/get-work-by-id.test.js

```diff
@@ -12,68 +13,96 @@ describe("Retrieve work by id", () => {
   describe("GET /works/{id}", () => {
     const { handler } = require("../../src/handlers/get-work-by-id");
 
-    it("retrieves a single work document", async () => {
-      mock
-        .get("/dc-v2-work/_doc/1234")
-        .reply(200, helpers.testFixture("mocks/work-1234.json"));
+    // it("retrieves a single work document", async () => {
```

Comment:
Are these tests no longer relevant or needed? If that's the case, let's remove them instead of commenting them out. If they need some refactoring to make them work with the new auth, let's do that instead.